cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -198,7 +241,20 @@ def base_operation(self) -> 'cirq.CircuitOperation':
 
     def replace(self, **changes) -> 'cirq.CircuitOperation':
         """Returns a copy of this operation with the specified changes."""
-        return dataclasses.replace(self, **changes)
+        kwargs = {
+            'circuit': self.circuit,
+            'repetitions': self.repetitions,
+            'qubit_map': self.qubit_map,
+            'measurement_key_map': self.measurement_key_map,
+            'param_resolver': self.param_resolver,
+            'repetition_ids': self.repetition_ids,
+            'parent_path': self.parent_path,
+            'extern_keys': self._extern_keys,
+            'use_repetition_ids': self.use_repetition_ids,
+            'repeat_until': self.repeat_until,
+        }
+        kwargs.update(changes)
```

Comment:
can use literal unpacking since later value overwrite earlier ones:
```suggestion
        kwargs = {
            'circuit': self.circuit,
            'repetitions': self.repetitions,
            'qubit_map': self.qubit_map,
            'measurement_key_map': self.measurement_key_map,
            'param_resolver': self.param_resolver,
            'repetition_ids': self.repetition_ids,
            'parent_path': self.parent_path,
            'extern_keys': self._extern_keys,
            'use_repetition_ids': self.use_repetition_ids,
            'repeat_until': self.repeat_until,
            **changes,
        }
```

Reply:
Done.